docs/data-feeds/historical-data.html

```diff
@@ -2,42 +2,42 @@
 layout: nodes.liquid
 section: ethereum
 date: Last Modified
-title: "Historical Price Data"
-permalink: "docs/historical-price-data/"
-whatsnext: {"API Reference":"/docs/price-feeds-api-reference/", "Contract Addresses":"/docs/reference-contracts/"}
+title: "Historical Data"
+permalink: "docs/data-feeds/historical-data/"
```

Comment:
I would put historical-data under docs/data-feeds/price-feeds/historical-data/

Rational:
- With price feeds, you can ether retrieve the current price or the historical data. Hence, both are path of the same functionality (price feeds)
- historical data uses the api-reference of price feed. Again, that shows that historical data is part of price feeds

Reply:
I was hoping to generecize this content, but I think I can do that in a followup PR. Moved Historical Data to the correct location.